test/integrations/destinations/marketo/dataDelivery/business.ts

```diff
@@ -0,0 +1,305 @@
+import { ProxyMetdata } from '../../../../../src/types';
+import { ProxyV1TestData } from '../../../testTypes';
+import { generateProxyV1Payload } from '../../../testUtils';
+
+const statTags = {
+  aborted: {
+    destType: 'MARKETO',
+    destinationId: 'dummyDestinationId',
+    errorCategory: 'network',
+    errorType: 'aborted',
+    feature: 'dataDelivery',
+    implementation: 'native',
+    module: 'destination',
+    workspaceId: 'dummyWorkspaceId',
+  },
+  retryable: {
+    destType: 'MARKETO',
+    destinationId: 'dummyDestinationId',
+    errorCategory: 'network',
+    errorType: 'retryable',
+    feature: 'dataDelivery',
+    implementation: 'native',
+    module: 'destination',
+    workspaceId: 'dummyWorkspaceId',
+  },
+  throttled: {
+    destType: 'MARKETO',
+    destinationId: 'dummyDestinationId',
+    errorCategory: 'network',
+    errorType: 'throttled',
+    feature: 'dataDelivery',
+    implementation: 'native',
+    module: 'destination',
+    workspaceId: 'dummyWorkspaceId',
+  },
+};
+
+export const proxyMetdata: ProxyMetdata = {
+  jobId: 1,
+  attemptNum: 1,
+  userId: 'dummyUserId',
+  sourceId: 'dummySourceId',
+  destinationId: 'dummyDestinationId',
+  workspaceId: 'dummyWorkspaceId',
+  secret: {},
+  dontBatch: false,
+};
+
+export const reqMetadataArray = [proxyMetdata];
+const params = {
+  destination: 'marketo',
+};
+
+const commonRequestParameters = {
+  JSON: {
+    action: 'createOrUpdate',
+    input: [
+      {
+        City: 'Tokyo',
+        Country: 'JP',
+        Email: 'gabi29@gmail.com',
+        PostalCode: '100-0001',
+        Title: 'Owner',
+        id: 1328328,
+        userId: 'gabi_userId_45',
+      },
+    ],
+    lookupField: 'id',
+  },
+  params,
+};
+
+export const testScenariosForV1API: ProxyV1TestData[] = [
```

Comment:
For Marketo there multiple response parsing scenarios where custom status code like 6XX and 7XX are parsed and application level errors are determined that is major use case for business scenarios we should test